modules/ossm-threescale-metrics.adoc

```diff
@@ -5,9 +5,13 @@
 
 [id="ossm-threescale-metrics_{context}"]
 = 3scale Adapter metrics
+
 The adapter, by default reports various Prometheus metrics that are exposed on port `8080` at the `/metrics` endpoint. These metrics provide insight into how the interactions between the adapter and 3scale are performing. The service is labeled to be automatically discovered and scraped by Prometheus.
 
-NOTE: There are incompatible changes in the 3scale Istio Adapter metrics since the previous releases in Service Mesh 1.x.
+[NOTE]
+====
+There are incompatible changes in the 3scale Istio Adapter metrics since the previous releases in Service Mesh 1.x.
```

Comment:
You could consider reversing the structure of this sentence, as follows:

>  Since the previous releases in Service Mesh 1.x, there are incompatible changes in the 3scale Istio Adapter metrics.

Reply:
Also, you could clarify what is meant by "since the previous releases". From that statement, I am unsure about which releases include the incompatibilities.